### ISSUE TITLE
A11y_WINFORMS_Winforms-Containers_GroupBox_Zoom:At 200% zoom the data under group box is not visible

### ISSUE THREAD

**Issue by Durgajyothi143** [dotnet/winforms#7353]
### .NET version

.NET Framework Version 4.8.04161
Operating System: Windows 11 Enterprise 21H2

### Did it work in .NET Framework?

No

### Issue description

GitHubTags:#A11yMAS;#A11yTCS;#.NET Core[WINFORMS]-Win32-June2022;#.NET Core;#A11ySev2;#WCAG1.4.4;#DesktopApp;#Win11;#Benchmark;#Zoom

**Actual:**

At 200% zoom the data under group box is not visible

**Expected:**

At 200% zoom the data under group box should be  visible

**User Impact:**

User with low vision will get difficulty to see the content.

### Steps to reproduce

1. Open Core testing solution in VS 2019 Int Preview
2. Right click on cs.proj file and click on Build.
3.Then run the application. 
4.Main Form screen would appear.
5.Navigate to Containers_Controls:Testing the controls under the containers Tab.
6.Winforms_Containers screen would appear.
7.Navigate to GroupBox section and verify all control.
 
**![A11y_WINFORMS_Winforms-Containers_GroupBox_Zoom #7353](https://user-images.githubusercontent.com/108055767/176091812-491b8130-4a0c-4c2e-9143-94e98b8a7033.png)**

**Comment by Olina-Zhang** [comment]
@merriemcgaw I verified this issue when the display scaling is at 200% and screen size is 1024x768, there is no scrollbar on form, and we can move mouse to see all form controls easily, please see following gif:
![200ZoomContainer](https://user-images.githubusercontent.com/26474449/176579222-ee9a147c-af4c-4c19-8f71-00b5a7222941.gif)

**Comment by merriemcgaw** [comment]
@Durgajyothi143 please retest with a screen size of 1280x1024 and the scaling (DPI scaling) at 200%. This is what I show, no content is out of the view of the window.
<img width="590" alt="Containers OK" src="https://user-images.githubusercontent.com/20996271/176795543-ac153c41-fb99-469d-beed-da3472635bc6.png">

**Comment by Durgajyothi143** [comment]
Verified the bug as per above comment issue is not reproducing. Hence closing the bug
![Presentation1](https://user-images.githubusercontent.com/108055767/179244092-23b8d9e0-ee49-4e5c-a08c-bdcb35b1ae14.gif)

**Comment by Tanya-Solyanik** [comment]
Closing per request from Samhitha K S (TATA CONSULTANCY SERVICES LTD)

Issue is fixed. Adding #Closed; tag.
